src/test/java/Homework22.java

```diff
@@ -0,0 +1,103 @@
+import org.openqa.selenium.WebDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.BasePage;
+import pages.HomePage;
+import pages.LoginPage;
+
+
+public class Homework22 extends BasePage {
+    public Homework22(WebDriver givenDriver) {
+        super(givenDriver);
```

Comment:
it seems you are confused about extending `BaseTest ` and `BasePage`.
`BaseTest `you will extend in test script, and `BasePage` will use extend in child page classes.

//HINT:
![image](https://github.com/testprojp/A47-QA-Automation-Framework/assets/44592718/b1911710-3d24-4465-b1c5-536736638076)